client/components/domains/reskin-side-explainer/index.jsx

```diff
@@ -32,29 +32,44 @@ class ReskinSideExplainer extends Component {
 				);
 				ctaText = translate( 'Use a domain I own' );
 				break;
+
+			case 'free-domain-only-explainer':
+				title = translate(
+					'Get a {{b}}free{{/b}} one-year domain registration with any paid annual plan.',
+					{
+						components: { b: <strong /> },
+					}
+				);
+				subtitle = translate(
+					'You can also choose to just start with a domain and add a site with a plan later on.'
+				);
+				break;
 		}
 
 		return { title, subtitle, ctaText };
 	}
 
 	render() {
 		const { title, subtitle, ctaText } = this.getStrings();
+		const { type } = this.props;
 
 		return (
 			/* eslint-disable jsx-a11y/click-events-have-key-events */
 			<div className="reskin-side-explainer">
 				<div className="reskin-side-explainer__title">{ title }</div>
 				<div className="reskin-side-explainer__subtitle">{ subtitle }</div>
-				<div className="reskin-side-explainer__cta">
-					<span
-						className="reskin-side-explainer__cta-text"
-						role="button"
-						onClick={ this.props.onClick }
-						tabIndex="0"
-					>
-						{ ctaText }
-					</span>
-				</div>
+				{ type !== 'free-domain-only-explainer' && (
```

Comment:
What do you think of changing this condition to `ctaText && (`? I think it would be more generic than checking for the type of the component - then if other component types also don't have a CTA, they wouldn't need to be added here